charon/src/expressions.rs

```diff
@@ -141,6 +141,14 @@ pub enum BinOp {
     Sub,
     /// Can overflow
     Mul,
+    /// Returns `(result, did_overflow)`, where `result` is the result of the operation with
+    /// wrapping semantics, and `did_overflow` is a boolean that indicates whether the operation
+    /// overflowed.
```

Comment:
"Note that on its side the unchecked [Add] operation panics in case of under/overflow."